src/server/verification/verificationAPI.js

```diff
@@ -39,69 +35,41 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
   app.post(
     '/verify/facerecognition',
     passport.authenticate('jwt', { session: false }),
-    upload.any(),
     wrapAsync(async (req, res, next) => {
+      const { sessionId, images } = req.body
+      const { user } = req
       const log = req.log
-      const { body, files, user } = req
-      log.debug({ user })
-      const sessionId = body.sessionId
+      let result = { ok: 0 }
+
+      if (user.identifier === undefined || sessionId === undefined || images === undefined) {
+        return res.status(400).send({ ok: 0, error: 'invalid input', isVerified: false })
+      }
       GunDBPublic.gun
         .get(sessionId)
```

Comment:
1.  .put({ idStarted: true });
2. assign session's ref to the const and re-use it

P.S. as session ref is used very often i propose to add it as the  method of GunDBPublic:

```
import { memoize } from 'lodash';

class GunDB implements StorageAPI {
// ...
session = memoize(sid => this.gun.get(sid));
// ..
}

// usage
const sessionRef = GunDBPublic.session(sessionId);
const isWhitelisted = await sessionRef.get('isWhitelisted').then();

Reply:
done